src/app/zap-templates/zcl/data-model/chip/rvc-run-cluster.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="ChangeToModeResponseStatus" type="ENUM8">
+    <cluster code="0x0054"/>
+    <item value="0x40" name="Battery Low"/>
+    <item value="0x41" name="Robot Stuck"/>
+    <item value="0x42" name="Bin Missing"/>
+    <item value="0x43" name="Bin Full"/>
+    <item value="0x44" name="Water Tank Missing"/>
+    <item value="0x45" name="Water Tank Lid Open"/>
+    <item value="0x46" name="Cleaning Pad Missing"/>
+  </enum>
+
+  <enum name="SemanticTags" type="ENUM16">
+    <cluster code="0x0054"/>
+    <item value="0x4000" name="Idle"/>
+    <item value="0x4001" name="Cleaning"/>
+  </enum>
+
+  <cluster>
+    <domain>General</domain>
+    <name>RVC Run</name>
+    <code>0x0054</code>
+    <define>RVC_RUN_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <description>This cluster is an alias of the Mode Select cluster which also defines a namespace for the running modes of the Robotic Vacuum Cleaner devices.</description>
+    <!-- Base data types -->
+    <attribute side="server" code="0x0000" define="MODE_DESCRIPTION" type="CHAR_STRING"                                     writable="false" optional="false" isNullable="false" length="64">Description</attribute>
```

Comment:
Needs ClusterRevision bits.

Reply:
Updated in [0d6b3d89be2beba6d443811cafdce3d8e88a11c1](https://github.com/project-chip/connectedhomeip/pull/26508/commits/0d6b3d89be2beba6d443811cafdce3d8e88a11c1)